load.php

```diff
@@ -1,9 +1,13 @@
 <?php
+/**
+ * Altis Search Module.
+ *
+ * @package altis/search
+ */
 
-// phpcs:ignore
-namespace Altis\Enhanced_Search;
+namespace Altis\Enhanced_Search; // phpcs:ignore
```

Comment:
Should only ignore relevant rule.

Reply:
Actually not sure which rule it is but I think it'll disappear anyway once namespaces are allowed in `load.php`